tox.ini

```diff
@@ -19,7 +19,7 @@ passenv = http_proxy https_proxy no_proxy SSL_CERT_FILE TOXENV CI TRAVIS TRAVIS_
 deps =
 extras = testing
 changedir = {toxinidir}/tests
-commands = pytest {posargs:--cov="{envsitepackagesdir}/tox" --cov-config="{toxinidir}/tox.ini"  --timeout=180 .}
+commands = pytest {posargs:--cov="{envsitepackagesdir}/tox" --cov-config="{toxinidir}/tox.ini"  --timeout=180 . -n auto}
```

Comment:
Does it actually make a difference? I tried this a while ago on my local machine (core i7 running on arch) and it was not faster at all, so I didn't bother to add it.

Reply:
On my machine with four core's it's ``74.31 seconds`` vs ``283.84 seconds`` so quite the difference.

Reply:
Seems it does not influence significantly the CI, but locally is very helpful.

Reply:
Oh well better than nothing then, if it speeds some local tests up. I guess at some point we should look into the way how we test tox anyway and see where the bottlenecks are.

Reply:
I plan to pick up in future the slow test batch:

```
10.73s call     tests/test_z_cmdline.py::test_test_usedevelop[.]
9.96s call     tests/test_z_cmdline.py::test_test_usedevelop[src]
9.78s call     tests/test_z_cmdline.py::test_usedevelop_mixed
8.21s call     tests/test_z_cmdline.py::test_toxuone_env
7.57s call     tests/test_z_cmdline.py::test_envtmpdir
7.01s call     tests/test_z_cmdline.py::test_different_config_cwd
6.41s call     tests/test_z_cmdline.py::test_notest
5.95s call     tests/test_z_cmdline.py::test_json
5.75s call     tests/test_z_cmdline.py::test_venv_special_chars_issue252
5.15s call     tests/test_z_cmdline.py::test_test_piphelp
5.10s call     tests/test_z_cmdline.py::test_verbosity[-vv]
4.93s call     tests/test_z_cmdline.py::test_env_VIRTUALENV_PYTHON
4.83s call     tests/test_z_cmdline.py::test_verbosity[-v]
4.77s call     tests/test_z_cmdline.py::test_exit_code[0]
4.77s call     tests/test_z_cmdline.py::test_alwayscopy_default
4.75s call     tests/test_z_cmdline.py::test_separate_sdist
4.72s call     tests/test_z_cmdline.py::test_exit_code[6]
4.70s call     tests/test_z_cmdline.py::test_verbosity[]
4.66s call     tests/test_z_cmdline.py::test_PYC
4.65s call     tests/test_z_cmdline.py::test_alwayscopy
4.62s call     tests/test_z_cmdline.py::test_envsitepackagesdir
4.54s call     tests/test_z_cmdline.py::test_empty_activity_ignored
4.41s call     tests/test_z_cmdline.py::test_package_install_fails
4.40s call     tests/test_z_cmdline.py::test_empty_activity_shown_verbose
4.37s call     tests/test_z_cmdline.py::test_developz
4.24s call     tests/test_z_cmdline.py::test_usedevelop
4.03s call     tests/test_z_cmdline.py::test_unknown_dep
3.46s call     tests/test_z_cmdline.py::test_skip_sdist
3.18s call     tests/test_z_cmdline.py::test_run_custom_install_command_error
0.84s call     tests/test_z_cmdline.py::TestSession::test_make_sdist
0.43s call     tests/test_config.py::TestCmdInvocation::test_colon_symbol_in_directory_name
0.37s call     tests/test_z_cmdline.py::test_unknown_interpreter
0.32s call     tests/test_z_cmdline.py::test_missing_env_fails
0.31s call     tests/test_z_cmdline.py::test_sdistonly
0.31s call     tests/test_z_cmdline.py::test_skip_platform_mismatch
0.30s call     tests/test_z_cmdline.py::test_envsitepackagesdir_skip_missing_issue280
0.28s call     tests/test_z_cmdline.py::test_unknown_interpreter_and_env
0.28s call     tests/test_z_cmdline.py::test_skip_unknown_interpreter_result_json
0.28s call     tests/test_z_cmdline.py::test_separate_sdist_no_sdistfile
0.27s call     tests/test_z_cmdline.py::test_skip_unknown_interpreter
0.26s call     tests/test_z_cmdline.py::TestSession::test_make_sdist_distshare
0.20s call     tests/test_config.py::TestCmdInvocation::test_showconfig_with_force_dep_version
0.18s call     tests/test_z_cmdline.py::test_tox_console_script
0.17s call     tests/test_z_cmdline.py::test_tox_quickstart_script
```
But for now this is lower hanging fruit.